File: source/api/recipients/update/_update.java
import java.util.HashMap;

import co.omise.model.Recipient;

Recipient recipient = Recipient.retrieve("recp_test_4z6p7e0m4k40txecj5o");
recipient.update(new HashMap<String, Object>() {
  {put("name", "somchai@prasert.com");}
  {put("bank_account", new HashMap<String, Object>() {
    {put("brand", "kbank");}
    {put("number", "1234567890");}
    {put("name", "SOMCHAI PRASERT");}
  });}
});

File: source/api/recipients/update/_update.py
import omise

omise.api_secret = 'skey_test_4xs8breq3htbkj03d2x'

recipient = omise.Recipient.retrieve("recp_test_511zkbdo44p30knf2lc")
recipient.update(
    email='somchai@prasert.com',
    bank_account=dict(
        brand='kbank',
        number='1234567890',
        name='SOMCHAI PRASERT'
    )
)